spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
+export PATH="/usr/local/pyload/bin:$PATH"
+
+case $1 in
+    start)
+        if [ -f /usr/syno/etc/ssl/ssl.crt/server.crt -a \
```

Comment:
Maybe I'm wrong, but SSL certificates are created by DSM only when necessary. If you turn off SSL in DSM, they're deleted (I have to recheck it, but I think it is so) => Pyload doesn't start. If you turn SSL on, then there is high probability you want it also in pyload.